os-bpmsuite-executionserver/added/launch/bpmsuite-executionserver.sh

```diff
@@ -66,6 +78,29 @@ function configure_controller_access {
     fi
 }
 
+function configure_router_access {
+    local routerService="${KIE_SERVER_ROUTER_SERVICE}"
+    routerService=${routerService^^}
+    routerService=${routerService//-/_}
+    # host
+    local kieServerRouterHost="${KIE_SERVER_ROUTER_HOST}"
+    if [ "${kieServerRouterHost}" = "" ]; then
```

Comment:
I think this won't work in case the `${kieServerRouterHost}` is actually empty. Bash will fail with something like `[: =: unary operator expected`. Using `if [ -z ${kieServerRouterHost} ];` (which checks is the length of the param is zero) should fix this.